api/DedicatedWorkerGlobalScope.json

```
@@ -85,7 +85,7 @@
               "version_added": true
```

Another mirrorable entry

```suggestion
              "version_added": "1.0"
```